templates/adguardhome/index.ts

```diff
@@ -0,0 +1,31 @@
+import { Output, Services } from "~templates-utils";
+import { Input } from "./meta";
+
+export function generate(input: Input): Output {
+  const services: Services = [];
+
+  services.push({
+    type: "app",
+    data: {
+      projectName: input.projectName,
+      serviceName: input.appServiceName,
+      source: {
+        type: "image",
+        image: input.appServiceImage,
+      },
+      proxy: {
+        port: 80,
+        secure: true,
+      },
+      mounts: [
+        {
+          type: "volume",
+          name: "data",
+          mountPath: "/opt/adguardhome/",
+        },
+      ],
+    },
+  });
```

Comment:
is it enough to expose this on port 80? don't you need to expose other ports as well?

Reply:
cant find a single thing using advanced ports

Reply:
fixing right NOW

Reply:
fixed